pkg/graph/resolvers/system_intake_status_admin_test.go

```diff
@@ -0,0 +1,387 @@
+package resolvers
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/cmsgov/easi-app/pkg/models"
+)
+
+type testSystemIntakeAdminStatusType struct {
+	testCase       string
+	intake         models.SystemIntake
+	expectedStatus models.SystemIntakeStatusAdmin
+	expectError    bool
+}
+
+// TestSystemIntakeStatusAdminGet tests the possible permutations for the systemIntakeStatusAdminGet method.
+// It doesn't use the testify suit, as this is not dependant on anything besides the resolver.
```

Comment:
It might be helpful to add a section to `docs/local_testing.md` on how to run individual tests when they're not part of a testify suite, it took me a moment to figure that out. Looks like the command is `go test -race <package name> -run <test name>`, i.e. `go test -race "github.com/cmsgov/easi-app/pkg/graph/resolvers" -run TestSystemIntakeStatusAdminGet`.

Reply:
Thanks Dylan, this could also be added the testify suite if you think it's better. I just liked the simplicity of being able to run the suite without needing to spin up the whole suite (it's faster for developing). I'll add to `local_test.md` regardless, because I think there are some other tests not using the testify suite.

Reply:
I'm fine keeping it out of the testify suite (especially since that suite needs the database running to spin up); I want to have the instructions for running a specific test documented, though.